<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>provideFile_widget</class>
 <widget class="QWidget" name="provideFile_widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>546</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>150</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="main_vertLayout">
     <item>
      <widget class="QFrame" name="frame">
       <property name="styleSheet">
        <string notr="true">QFrame#frame{
     border: 1px solid #FF17365D;
     border-bottom-left-radius: 15px;
     border-bottom-right-radius: 15px;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="PyDMLabel" name="TitleLabel">
          <property name="toolTip">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
 qproperty-alignment: AlignCenter;
 border: 1px solid #FF17365D;
 border-top-left-radius: 15px;
 border-top-right-radius: 15px;
 background-color: #FF17365D;
 padding: 5px 0px;
 color: rgb(255, 255, 255);
 max-height: 25px;
 font-size: 14px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="PyDMLabel" name="HeaderLabel">
            <property name="toolTip">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="fileDialog_layout"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
